react/components/MicroData.js

```diff
@@ -1,89 +1,140 @@
 import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
-import { pathOr, path, map, sort, compose, head, split } from 'ramda'
-
-const ITEM_AVAILABLE = 100
-
-const lowestPriceInStockSeller = (item) => {
-  if (item.sellers.length) {
-    return sort((itemA, itemB) => itemA.commertialOffer && itemA.commertialOffer.AvailableQuantity > 0
-      ? itemB.commertialOffer && itemB.commertialOffer.AvailableQuantity > 0
-        ? itemA.commertialOffer.Price - itemB.commertialOffer.Price
-        : -1
-      : -1,
-    item.sellers)[0]
+import { pathOr, path, sort, last, flatten } from 'ramda'
```

Comment:
This code was copy and pasted of https://github.com/vtex-apps/structured-data/blob/master/react/Product.js